final-final-errors

```diff
@@ -0,0 +1,47 @@
+tasks/administer-cluster/reconfigure-kubelet/index.html
+  hash does not exist --- tasks/administer-cluster/reconfigure-kubelet/index.html --> #understanding-node-status-config-error-messages
+tasks/access-application-cluster/access-cluster/index.html
+  hash does not exist --- tasks/access-application-cluster/access-cluster/index.html --> #accessing-the-cluster-api
+setup/production-environment/tools/kubespray/index.html
+  target is a directory, href lacks trailing slash --- setup/production-environment/tools/kubespray/index.html --> ../kops
+setup/production-environment/tools/kubeadm/index.html
+  hash does not exist --- setup/production-environment/tools/kubeadm/index.html --> #verify-the-mac-address-and-product-uuid-are-unique-for-every-node
+setup/production-environment/tools/kubeadm/install-kubeadm/index.html
+  hash does not exist --- setup/production-environment/tools/kubeadm/install-kubeadm/index.html --> #verify-the-mac-address-and-product-uuid-are-unique-for-every-node
+contribute/review/for-approvers/index.html
+  hash does not exist --- contribute/review/for-approvers/index.html --> #add-and-remove-labels
+concepts/scheduling-eviction/scheduling-framework/index.html
+  hash does not exist --- concepts/scheduling-eviction/scheduling-framework/index.html --> #frameworkhandle
+concepts/storage/persistent-volumes/index.html
+  target does not exist --- concepts/storage/persistent-volumes/index.html --> docs/concepts/storage/persistent-volumes/#raw-block-volume-support
+concepts/services-networking/dns-pod-service/index.html
+  hash does not exist --- concepts/services-networking/dns-pod-service/index.html --> #pod-s-dns-config
+concepts/workloads/pods/init-containers/index.html
+  hash does not exist --- concepts/workloads/pods/init-containers/index.html --> #what-s-next
+concepts/extend-kubernetes/operator/index.html
+  hash does not exist --- concepts/extend-kubernetes/operator/index.html --> #what-s-next
+concepts/cluster-administration/flow-control/index.html
+  hash does not exist --- concepts/cluster-administration/flow-control/index.html --> #what-s-next
+tutorials/stateless-application/guestbook-logs-metrics-with-elk/index.html
+  target is a directory, href lacks trailing slash --- tutorials/stateless-application/guestbook-logs-metrics-with-elk/index.html --> ../guestbook
+  target is a directory, href lacks trailing slash --- tutorials/stateless-application/guestbook-logs-metrics-with-elk/index.html --> ../guestbook
+  target is a directory, href lacks trailing slash --- tutorials/stateless-application/guestbook-logs-metrics-with-elk/index.html --> ../guestbook
+  target is a directory, href lacks trailing slash --- tutorials/stateless-application/guestbook-logs-metrics-with-elk/index.html --> ../guestbook
+========================================================================
+✘✘✘ failed in 800.791996ms
+24 errors in 573 documents
```

Comment:
@celestehorgan I imagine you'd want to `.gitignore` this file?

Reply:
Delete it, more like ;)